src/plugins/azgenerator/TemplateAzureCliClientFactory.ts

```diff
@@ -16,7 +16,7 @@ export function GenerateAzureCliClientFactory(model: CodeModelAz) : string[] {
     output.push("    from azure.cli.core.commands.client_factory import get_mgmt_service_client");
     output.push("    from ..vendored_sdks." + model.PythonOperationsName + " import " + model.PythonMgmtClient);
 
-    if (model.Extension_ClientSubscriptionBound || model.Extension_ClientBaseUrlBound)
+    if (model.Extension_ClientSubscriptionBound != undefined || model.Extension_ClientBaseUrlBound != undefined)
```

Comment:
it's better to use IsUndefinedOrNull() check instead of =, it's always confusing and easy to make mistake for that kind of comparison in js. thx

Reply:
Fixed. Thanks